release.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs ? import ./nixpkgs.nix }: with pkgs;
+
+let
+  buildFlatpak = callPackage (fetchGit {
+    url = "https://github.com/serokell/nix-flatpak";
+    rev = "46a2aadf37981d6313621913cd2802debfb763fd";
```

Comment:
I'm not sure if I'm happy with abolishing the serokell-overlay fetch, changing to 3 different fetches. Maybe These should be in serokell-public-overlay if that still exists.

Reply:
It still exists. Guess that would work.